tests/Silex/Tests/ControllerResolverTest.php

```diff
@@ -24,6 +25,10 @@ class ControllerResolverTest extends \PHPUnit_Framework_TestCase
 {
     public function testGetArguments()
     {
+        if (Kernel::VERSION_ID >= 30100) {
+            $this->markTestSkipped('This code is not used anymore with Symfony 3.1+');
```

Comment:
no need to skip the test entirely. Just keep it and mark it as ``@group legacy``. This way, the class is tested even when you have 3.1 installed locally